osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -228,60 +232,50 @@ public ModSelectOverlay()
-                                                    Right = 20
-                                                }
+                                                Origin = Anchor.CentreLeft,
+                                                Anchor = Anchor.CentreLeft,
                                             },
-                                            new OsuSpriteText
+                                            new FillFlowContainer
                                             {
-                                                Text = @"Score Multiplier:",
-                                                Font = OsuFont.GetFont(size: 30),
-                                                Margin = new MarginPadding
-                                                {
-                                                    Top = 5,
-                                                    Right = 10
-                                                }
-                                            },
-                                            MultiplierLabel = new OsuSpriteText
-                                            {
-                                                Font = OsuFont.GetFont(size: 30, weight: FontWeight.Bold),
-                                                Margin = new MarginPadding
+                                                AutoSizeAxes = Axes.Both,
+                                                Spacing = new Vector2(footer_button_spacing / 2, 0),
+                                                Origin = Anchor.CentreLeft,
+                                                Anchor = Anchor.CentreLeft,
+                                                Children = new Drawable[]
                                                 {
-                                                    Top = 5
-                                                }
+                                                    new OsuSpriteText
+                                                    {
+                                                        Text = @"Score Multiplier:",
+                                                        Font = OsuFont.GetFont(size: 30),
+                                                        Origin = Anchor.CentreLeft,
+                                                        Anchor = Anchor.CentreLeft,
+                                                    },
+                                                    MultiplierLabel = new OsuSpriteText
+                                                    {
+                                                        Font = OsuFont.GetFont(size: 30, weight: FontWeight.Bold),
+                                                        Origin = Anchor.CentreLeft,
+                                                        Anchor = Anchor.CentreLeft,
+                                                        Width = 70, // to prevent footer from flowing when clicking mods
```

Comment:
can you explain the scenario where this happens? i can't repro.

Reply:
- Change UI scale just before the footer flows.
- Click on a multiplier decrease mod.
![vKCNf3ACmb](https://user-images.githubusercontent.com/35318437/80446263-3d83ae00-88cb-11ea-8d2b-e553e9716cff.gif)

This is because the multiplier number can increase/decrease width based on the numbers.

Reply:
Aha, I see. Will reword the comment slightly.